### ISSUE TITLE
Create Java code snippets for bitfields

### ISSUE THREAD

**Issue by sav-norem** [redis/jedis#3628]
The redis.io webpage for [bitfields](https://redis.io/docs/data-types/bitfields/) needs the Java example as well. The python equivalent can be used to copy the setup and can be found [here](https://github.com/redis/redis-py/pull/3002/files#diff-75621ec51da32e7f657243e238496703618ccfb882d0eac5b7c61ed400e1830f)

The first line of the file needs to be the ID for the example - # EXAMPLE: bitfield_tutorial

Each separate section is delimited with a tag that you will need to copy exactly for it to appear in the correct place, ie.
```
# STEP_START bf
{code}                                    
# STEP_END

# HIDE_START, # HIDE_END, # REMOVE_START, and # REMOVE_END can be used for debugging and deleting keys. Hide will hide until a user expands the example, remove will only be seen in the source code and will never appear on the webpage.
```

@dwdougherty and I will be reviewing and tracking these issues - feel free to tag us with any questions

**Comment by Ranjeet0611** [comment]
@sav-norem @dwdougherty I have gone through the python example of bitfields but there in line 13 it is returning object where we can then call set and execute method.
But in java it is returning list of long so i will not be able to call set and execute method on arraylist.

**Comment by sazzad16** [comment]
Resolved by #3894
